File: SpeedTestGameClient/core/src/main/java/si/um/feri/speedii/dao/MobileTowerCRUD.java
package si.um.feri.speedii.dao;

import java.util.List;

import si.um.feri.speedii.classes.MobileTower;

public interface MobileTowerCRUD extends DaoCrud<MobileTower> {

    List<MobileTower> getByConfirmed(boolean status);

    boolean toggleConfirm(MobileTower obj);

    boolean insertMany(List<MobileTower> list);
}

File: MobileApp/app/speedtest/src/main/java/sosteric/pora/speedtest/dao/MobileTowerCRUD.kt
package dao

import MobileTower

interface MobileTowerCRUD: DaoCrud<MobileTower>{
    fun getByConfirmed(status: Boolean) : List<MobileTower>
    fun toggleConfirm(obj: MobileTower) : Boolean
    fun insertMany(list: List<MobileTower>): Boolean
}